src/components/opening-hours/opening-hours.vue

```diff
@@ -68,6 +76,49 @@ export default {
   },
   computed: {
     ...mapState(['isMobile']),
+    timeSlots() {
+      const { openingHoursSpace, openingHoursBuilding } = this
+      const newOpeningHours = openingHoursSpace.map((openingHour, index) => {
+        const buildingStart = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][0] : []
+        const buildingEnd = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][1] : []
+        const newTimeslots = []
```

Comment:
Shouldn't this be an object? If I'm correct we're only setting and getting the `day` and `times` property and not using its iterable features.